prometheus_client/gc_collector.py

```diff
@@ -3,79 +3,44 @@
 from __future__ import unicode_literals
 
 import gc
-import os
-import time
 
-from .metrics import Histogram
+from .metrics_core import CounterMetricFamily
 from .registry import REGISTRY
 
 
 class GCCollector(object):
     """Collector for Garbage collection statistics."""
 
-    def __init__(self, registry=REGISTRY, gc=gc):
-        # To work around the deadlock issue described in
-        # https://github.com/prometheus/client_python/issues/322,
-        # the GC collector is always disabled in multiprocess mode.
-        if 'prometheus_multiproc_dir' in os.environ:
+    def __init__(self, registry=REGISTRY):
+        if not hasattr(gc, 'get_stats'):
             return
+        registry.register(self)
 
-        if not hasattr(gc, 'callbacks'):
-            return
-
-        collected = Histogram(
-            'python_gc_collected_objects',
+    def collect(self):
+        collected = CounterMetricFamily(
+            'python_gc_objects_collected',
```

Comment:
Counters should end in _total

Reply:
`CounterMetricFamily` will add it autocratically.  You can see it in tests https://github.com/prometheus/client_python/pull/371/files#diff-b8616fecc6de945cdf630d3ad09aebb8R26

Reply:
Ah yes, I'd forgotten I'd already added that.